src/components/recipe.js

```diff
@@ -12,28 +12,28 @@ class recipe extends React.Component {
   }
 
   render() {
-    const { recipes } = this.props;
-    const singleRecipe = recipes.map(recipe => {
+    
       return (
-        <div className="recipe" key={recipe.id}>
+        <div className="recipe">
           <div className="recipe-text-main">
-            <h3 className="recipe-title">{recipe.title}</h3>
+            <h3 className="recipe-title">{this.props.title}</h3>
             <p>
-              <i class="fa fa-history"></i>
-              {recipe.readyInMinutes} minutes
+              <i className="fa fa-history"></i>
+              {this.props.min} minutes
```

Comment:
{this.props.min} -> I think the 'min' property might need to be 'readyInMinutes' which is what Spoonacular returns.